sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -1005,12 +1005,12 @@ public void beginRecognizeIDDocumentFromUrl() {
                         }
                     }
 
-                    FormField countryFormField = recognizedFields.get("Country");
-                    if (countryFormField != null) {
-                        if (FieldValueType.STRING == countryFormField.getValue().getValueType()) {
-                            String country = countryFormField.getValue().asCountry();
-                            System.out.printf("Country: %s, confidence: %.2f%n",
-                                country, countryFormField.getConfidence());
+                    FormField countryRegionFormField = recognizedFields.get("CountryRegion");
+                    if (countryRegionFormField != null) {
+                        if (FieldValueType.STRING == countryRegionFormField.getValue().getValueType()) {
+                            String countryRegion = countryRegionFormField.getValue().asCountryRegion();
+                            System.out.printf("Country region: %s, confidence: %.2f%n",
```

Comment:
I would stick to the same rendering as the service uses here, or expand to "Country or region".

```suggestion
                            System.out.printf("CountryRegion: %s, confidence: %.2f%n",
```